src/pivotal-ui/components/meta/package.json

```diff
@@ -4,5 +4,5 @@
     "@npmcorp/pui-css-bootstrap": "^2.0.1",
     "@npmcorp/pui-css-typography": "^4.0.0"
   },
-  "version": "1.3.3"
+  "version": "1.3.4"
```

Comment:
1.3.3 went through with another change. I didn't bump a version with my last PR (they were unrelated but parallel on the same package)